Zc-specification/Zc.adoc

```diff
@@ -284,6 +284,27 @@ The Zcmt extension depends on the <<Zca>> and Zicsr extensions.
 
 |===
 
+[#Zc_formats]
+=== Zc instruction formats
+
+[%header,cols="2,3,2,1,1,1,1,1,1,1,1,1,1"]
+|=====================================================================
+| Format | instructions       | 15:10  | 9  | 8 | 7   | 6 | 5 | 4   | 3 | 2  | 1 | 0
+| CLBU   | c.lbu              | funct6 3+| rs1'    2+| uimm    3+| rd'    2+| op

```

Comment:
Maybe call this `CLB` for symmetry with `CLH`?  (Yes, I realize there's no `c.lb` instruction.)